logback.xml

```diff
@@ -16,6 +16,7 @@
   -->
 
 <configuration>
+    <property name="LOGS_DIR" value="/var/lib/tomcat/logs" />
```

Comment:
IIRC we said we don't want this (we'd rather change `LOGS_DIR` to default to `.` instead of `./logs`)?

Reply:
I thought we also said, i tried and it dod not work :-)

Reply:
Now, you make me doubt, i guess i am gonna have to try again...

Reply:
Ack - just removed it.